modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -189,13 +189,15 @@ It is the admin's responsibility to make the files persistent.
 To inspect the volumes run:
 [source,docker]
 ----
-docker volume ls | grep ownclouddockerserver
+docker volume ls
```

Comment:
We can leave a `grep` for `files` here as only the prefix will change, the name `files` is given in the compose file.

```suggestion
docker volume ls | grep files
```